src/views/view.c

```diff
@@ -1079,6 +1079,10 @@ dt_view_surface_value_t dt_view_image_get_surface(int imgid, int width, int heig
   dt_mipmap_cache_release(darktable.mipmap_cache, &buf);
   if(rgbbuf) free(rgbbuf);
 
+  dt_print(DT_DEBUG_LIGHTTABLE,
+           "[dt_view_image_get_surface]  id %i, dots %ix%i, mip %ix%i, surf %ix%i created in %0.04f sec\n", imgid,
+           width, height, buf_wd, buf_ht, img_width, img_height, dt_get_wtime() - tt);
```

Comment:
this dt_get_wtime() should be also a variable outside of this call to avoid the evaluation when not needed.